docs/modules/ROOT/pages/cops_lint.adoc

```diff
@@ -1147,6 +1149,11 @@ items.each { |item| }
 
 # good
 items.each { |item| puts item }
+
+allow(subject).to receive(:callable).and_return(-> {})
+
+lambda do
+end
```

Comment:
Ah, this was a bad example I wrote 💦  IMHO, the case where empty `{}` is often used is in the one-liner.
E.g. `-> {}` or `lambda {}`.
So, I'd like to hear other opinions as to whether it should be allowed even in the case of multi-liner.

Reply:
I think it makes sense to allow both, because `Style/Lambda` exists and is configurable to both kinds.